infra/terraform/dev/deploy/launch.tf

```
@@ -0,0 +1,35 @@
+data "template_cloudinit_config" "config" {
+  gzip          = false
+  base64_encode = true
+
+  part {
+    content_type = "text/x-shellscript"
+    content      = <<EOT
+#!/bin/bash
```

Always recommend setting these on scripts to make soft-warnings into errors.

* `set -e` -- fail if any command does
* `set -u` -- fail if a bash variable is unset when used

```suggestion
#!/bin/bash
set -eu
```

---

Incorporated in next commit